application/common/application_data.h

```diff
@@ -200,6 +203,11 @@ class ApplicationData : public base::RefCountedThreadSafe<ApplicationData> {
   // The source the application was loaded from.
   SourceType source_type_;
 
+  // The optional window size.
+  int window_width_;
```

Comment:
it would be better to have some pair type for both width and height, gfx::Size maybe?